cmd/csaf_aggregator/config.go

```diff
@@ -36,12 +36,13 @@ type provider struct {
 	Name   string `toml:"name"`
 	Domain string `toml:"domain"`
 	// Rate gives the provider specific rate limiting (see overall Rate).
-	Rate       *float64  `toml:"rate"`
-	Insecure   *bool     `toml:"insecure"`
-	Categories *[]string `toml:"categories"`
+	Rate         *float64  `toml:"rate"`
+	Insecure     *bool     `toml:"insecure"`
+	WriteIndices *bool     `toml:"write_indices"`
+	Categories   *[]string `toml:"categories"`
 	// ServiceDocument incidates if we should create a service.json document.
-	ServiceDocument *bool `toml:"create_service_document"`
-	WriteIndices    *bool `toml:"write_indices"`
+	ServiceDocument     *bool                    `toml:"create_service_document"`
+	AggregatoryCategory *csaf.AggregatorCategory `toml:"category"`
```

Comment:
There seems to be a potential for confusion as a provider strictly does not have an AggregatoryCategory, because a provider is not an aggregator. I suggest to use a different name and type for this configuration option, what about bool and something about to_be_mirrored or so?